README.md

```diff
@@ -236,3 +283,8 @@ Not yet, but this is planned.
 ### Does beehive-flow work with any types of package other than NPM?
 
 Not at this stage.
+
+### Why does the Jenkinsfile example above run `beehive-flow publish` via npx instead of yarn?
+
+When `npm publish` is run from within yarn, yarn sets an environment variable to point to the yarn repo URL, not the NPM one. 
+This causes errors for beehive-flow's publishing phase.
```

Comment:
This really is an issue with our CI infrastructure rather than beehive flow, as it's just missing the credentials to authenticate against the yarn repository (ie I'm pretty sure you'd need to run `yarn login` locally to have it work).

Reply:
The other option would be to make sure to include the `publishConfig` registry configuration in the package.json.

Reply:
Should I omit this bit from the readme, then?

Reply:
Maybe just remove the second sentence?